fjord/feedback/views.py

```diff
@@ -72,11 +72,18 @@ def thanks(request):
             pass
 
     if feedback:
-        product = feedback.product
+        product_db_name = feedback.product
         suggestions = get_suggestions(feedback, request)
 
-    return render(request, 'feedback/thanks.html', {
-        'product': product,
+    product = models.Product.objects.get(db_name=product_db_name)
+
+    template = 'feedback/thanks.html'
+
+    if product.slug in PRODUCT_OVERRIDE:
+      template = PRODUCT_OVERRIDE[product.slug]['thanks_template']
```

Comment:
The fennec input and thank you pages have their own style (see screenshots here: https://docs.google.com/document/d/1yROux6xW2vzjpV9OYx9ppOtdYm-Mu3rApjZ3C-8xGLg/edit#bookmark=id.k0hw2dxlnutc) and content; they don't take Input's standard style or content.

And this project's ultimate intent is to make optimizing them easier, which requires that they be isolated from the generic templates.

Regarding two/four spaces, that's my bad. My editor doesn't switch indent styles automatically. I will definitely run code through flake8 before final PR, so you can probably ignore nits for now. :)